src/systemcmds/param/param.cpp

```diff
@@ -783,7 +821,62 @@ do_set(const char *name, const char *val, bool fail_on_not_found)
 }
 
 static int
-do_compare(const char *name, char *vals[], unsigned comparisons, enum COMPARE_OPERATOR cmp_op, enum COMPARE_ERROR_LEVEL err_level)
+do_set_custom_default(const char *name, const char *val)
+{
+	param_t param = param_find(name);
```

Comment:
You can use `param_find_no_notification`, so it's not marked as used, and the check in https://github.com/PX4/PX4-Autopilot/pull/16796/commits/ee31f7b2bf8e30af61144ad07daf25041233ef22#diff-55e7ab8e00d96dde91d94330ed29f1e94fd797e7f9cc5b8a2945602d33b70895R931 will not notify.

Reply:
Yes of course, thanks.